pyproject.toml

```diff
@@ -60,66 +60,66 @@ matplotlib = "pandas:plotting._matplotlib"
 [project.optional-dependencies]
 test = ['hypothesis>=6.84.0', 'pytest>=7.3.2', 'pytest-xdist>=3.4.0']
 pyarrow = ['pyarrow>=10.0.1']
-performance = ['bottleneck>=1.3.6', 'numba>=0.56.4', 'numexpr>=2.8.4']
-computation = ['scipy>=1.10.0', 'xarray>=2022.12.0']
-fss = ['fsspec>=2022.11.0']
-aws = ['s3fs>=2022.11.0']
-gcp = ['gcsfs>=2022.11.0']
-excel = ['odfpy>=1.4.1', 'openpyxl>=3.1.0', 'python-calamine>=0.1.7', 'pyxlsb>=1.0.10', 'xlrd>=2.0.1', 'xlsxwriter>=3.0.5']
+performance = ['bottleneck>=1.3.6', 'numba>=0.59.0', 'numexpr>=2.9.0']
+computation = ['scipy>=1.12.0', 'xarray>=2024.1.1']
+fss = ['fsspec>=2024.2.0']
+aws = ['s3fs>=2024.2.0']
+gcp = ['gcsfs>=2024.2.0']
+excel = ['odfpy>=1.4.1', 'openpyxl>=3.1.2', 'python-calamine>=0.1.7', 'pyxlsb>=1.0.10', 'xlrd>=2.0.1', 'xlsxwriter>=3.2.0']
 parquet = ['pyarrow>=10.0.1']
 feather = ['pyarrow>=10.0.1']
 hdf5 = [# blosc only available on conda (https://github.com/Blosc/python-blosc/issues/297)
         #'blosc>=1.20.1',
         'tables>=3.8.0']
-spss = ['pyreadstat>=1.2.0']
+spss = ['pyreadstat>=1.2.6']
 postgresql = ['SQLAlchemy>=2.0.0', 'psycopg2>=2.9.6', 'adbc-driver-postgresql>=0.10.0']
-mysql = ['SQLAlchemy>=2.0.0', 'pymysql>=1.0.2']
+mysql = ['SQLAlchemy>=2.0.0', 'pymysql>=1.1.0']
 sql-other = ['SQLAlchemy>=2.0.0', 'adbc-driver-postgresql>=0.10.0', 'adbc-driver-sqlite>=0.8.0']
-html = ['beautifulsoup4>=4.11.2', 'html5lib>=1.1', 'lxml>=4.9.2']
+html = ['beautifulsoup4>=4.12.3', 'html5lib>=1.1', 'lxml>=4.9.2']
 xml = ['lxml>=4.9.2']
-plot = ['matplotlib>=3.6.3']
-output-formatting = ['jinja2>=3.1.2', 'tabulate>=0.9.0']
+plot = ['matplotlib>=3.8.3']
+output-formatting = ['jinja2>=3.1.3', 'tabulate>=0.9.0']
 clipboard = ['PyQt5>=5.15.9', 'qtpy>=2.3.0']
-compression = ['zstandard>=0.19.0']
+compression = ['zstandard>=0.22.0']
 timezone = ['pytz>=2023.4']
 all = ['adbc-driver-postgresql>=0.10.0',
```

Comment:
Slightly tangential but the adbc driver libraries are all released at the same time. Not sure how we got a minimum for SQLite of 0.8.0 and postgresql at 0.10.0, but should probably bump the former to match the latter

Reply:
Sure thing. I tried at least moving these installs to conda-forge in https://github.com/pandas-dev/pandas/pull/60655/commits/39cdaf19f3e0d461f8f1c016c450b466a81f31a7 but I was getting some `openssl` dependency conflicts. I'll try again with bumping these version in the coming weeks

Reply:
There is still an open issue on windows for using conda-forge, so I'd suggest punting that until fixed upstream. See https://github.com/apache/arrow-adbc/issues/1149

Reply:
Hmm I see that that the conda forge packages are `noarch` now so I think they should work on Windows now? https://anaconda.org/conda-forge/adbc-driver-postgresql/files?page=2

Reply:
Perhaps that worked for early versions of arrow-adbc (I see those go up to 0.5.0) but at the moment there is no way for these to be architecture independent